### PR TITLE
arrange_contents: fix calling without the `path` parameter

### PR DESCRIPTION
**Pull request by mistydemeo**
`DEFAULT_ARRANGE_PATH` is not base64-encoded; trying to decode it raises an exception.

### CODE REVIEW

File: src/dashboard/src/components/filesystem_ajax/views.py [hunk 1]
```diff
@@ -114,13 +114,23 @@ def contents(request):
...
-    if not base_path.endswith('/'):
-        base_path += '/'
+    path = request.GET.get('path')
+    if path is not None:
+        try:
+            base_path = base64.b64decode(path)
+        except TypeError:
+            response = {
+                'success': False,
+                'message': 'Could not base64-decoded provided path: ' + path,
```
Comment (Hwesta) [inline src/dashboard/src/components/filesystem_ajax/views.py]: Use % or .format or str(), since `+` will fail if path is not a basestring.

Comment (mistydemeo) [inline src/dashboard/src/components/filesystem_ajax/views.py]: Fixed! Too much time in JS land...

File: src/dashboard/src/components/filesystem_ajax/views.py [hunk 2]
```diff
@@ -114,13 +114,23 @@ def contents(request):
...
+            }
+            helpers.json_response(response, status_code=400)
```
Comment (Hwesta) [inline src/dashboard/src/components/filesystem_ajax/views.py:126]: https://docs.djangoproject.com/en/1.8/ref/request-response/#jsonresponse-objects :o Probably not for this PR but I'd like to replace `helpers.json_response` with `JsonResponse`

Comment (mistydemeo) [inline src/dashboard/src/components/filesystem_ajax/views.py:126]: Yeah, in the future let's make that change all at once.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**